tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -661,6 +695,48 @@ public String getVersion()
         return version;
     }
 
+    private String getPluginFolder( String type )
```

Comment:
Seems like this type of code exists in other places, can you do a quick check and see if we can reuse something else here instead of having this code duplicated?